guides/process/scheduling/how_to_timesheet.md

```diff
@@ -44,6 +44,20 @@ You should allocate your time to a project when you’ve spent 2 hours or more.
 
 We use Kimble to complete timesheets. All resources on how to complete and approve timesheets are in the [Knowledge base](https://sites.google.com/madetech.com/signpost/home/kimble-resources?authuser=0).
 
+### Why we need you to do timesheets
+
+Timesheet data directly drives our client invoicing. If you don't timesheet, the business can't invoice and we lose revenue which has an impact on all of us.
+Timesheet data also creates wider business-critical data about utilisation, client investments and time spent on other activities that's needed for reporting and planning.
+
+### Why we need you to do timesheets weekly
+Whilst we invoice our clients monthly it's vitally important that the data driving these invoices is accurate as clients do actually check, after all nobody likes being overcharged for anything. Weekly timesheets ensure we capture time spent while it's fresh in our memories.
```

Comment:
As above - would suggest a wording amend to:
'Whilst we invoice our clients monthly it's vitally important that the data driving these invoices is accurate at a week by week level. Many clients do check the individual items driving an invoice to validate that they've been correctly charged by us. Weekly timesheets ensure we capture time spent while it's fresh in our memories.

Reply:
Updated